.eslintrc.json

```diff
@@ -18,8 +18,7 @@
               }
             ]
           }
-        ],
-        "@nx/workspace-no-terminal-popups": "error"
```

Comment:
maybe not a good place, but my initial intent was to cover both `ts` and `js` files

Reply:
I reverted my change. I just moved it under TS overrides since the codebase is not using JS, but we don't know what the future holds ;)